### PR TITLE
Fix #320 Auto-assign reviewers for a PR when created

### LINKED ISSUE #320: Auto-assign reviewers for a PR when created, and give clearer instructions to contributors on how to make that assignment happen.
**Is your feature request related to a problem? Please describe.**

I noticed that this PR got auto-closed recently: https://github.com/oppia/oppia/pull/18065 . On examining it, it seems that there were two process failures:

(a) When the PR was opened, Oppiabot didn't automatically assign the codeowners (should be @gp201 and @seanlip)
(b) When Oppiabot asked the author to assign the required reviewers for the PR, it didn't explain how to do it (both in terms of figuring out who those reviewers are, and explaining how to send a message to Oppiabot to do the assignment since new contributors cannot assign reviewers directly).

I'm not sure if the original contributor is actually tracking the PR, but AFAICT they never seemed to reach out, though they did keep merging develop into the PR. Maybe they didn't know who to reach out to.

In any case, this flow seems to be suboptimal and has resulted in a "waste" of a PR, as well as a potentially frustrating experience for the contributor.

**Describe the solution you'd like**

Ideally, Oppiabot does (a), but we should also update the message to do (b).

Consider also making it clearer to folks that they should reach out to the dev workflow team or on Discussions if their PR hasn't seen any action after 2 days. Maybe Oppiabot can ping the dev workflow team directly.

**Describe alternatives you've considered**

We could do a subset of the above, depending on what is feasible and makes sense.

### PR DESCRIPTION
**Pull request by 462702985**
https://github.com/oppia/oppiabot/assets/85102180/7ce79ffb-8c39-4344-a5f8-7da4c99f6ca4

## Explanation

Fix #320: wait 30s before oppiabot assign reviewers. github will have time to collect reviewers first.
## Checklist
- [x] I have successfully deployed my own instance of Oppiabot.
  - You can find instructions for doing this [here](https://github.com/oppia/oppiabot/wiki/Deploying-your-own-instance-of-the-oppiabot).
- [x] I have manually tested all the changes made in this PR following the [manual tests matrix](https://github.com/oppia/oppiabot/wiki/Manual-Tests-Matrix).

### CODE REVIEW

**Comment by U8NWXD** [comment]
@seanlip who should review this PR? We need one approving review from someone with write access, but I don't know who those people are

**Review by gp201 — APPROVED** [review]
LGTM

**Comment by gp201** [comment]
@vojtechjelinek PTAL

File: lib/checkPullRequestLabels.js
Comment (vojtechjelinek) [inline lib/checkPullRequestLabels.js:45]: Hmm, lets put a bit less, I think 10 seconds should be enough, 30 seconds wait can make the oppiabot timeout and not finish.

**Review by vojtechjelinek — CHANGES_REQUESTED** [review]
Thanks! Just one comment.

**Comment by vojtechjelinek** [comment]
@462702985 When you fix something please reply to all the review comments and ping the reviewer again so that they know to do a review again.

**Review by vojtechjelinek — APPROVED** [review]
LGTM!

**Pull request merged**
